### DISCUSSION TITLE
How to know the `nvimdots` version

### DISCUSSION THREAD

**Discussion by SimFG** [ayamir/nvimdots#723] [Q&A] [answered]
I am curious how to know the version of `nvimdot`, or can I get the latest `nvimdot` only by syncing the plugins?

**Comment by charliie-dev** [comment]
That's a good suggestion. We can include a version file in the repo.

**Comment by ayamir** [comment, marked as answer]
`git tag` and `git checkout tags/vx.x.x`
<img width="546" alt="image" src="https://user-images.githubusercontent.com/61657399/236777482-5b28b3cb-93dd-4d82-8927-4b84633c0fc5.png">
About the plugins' version, `:Lazy restore`

**Comment by SimFG** [comment]
![image](https://user-images.githubusercontent.com/21985684/236778845-015299d4-db15-4ec7-9c88-b07864a93d23.png)
My git history has only one record and there are some lazy-lock.json changes

**Reply by ayamir** [reply]
Say `n` when encounter this prompt:
https://github.com/ayamir/nvimdots/blob/3a150ef17f93bf3040e124bf47a778f276820be1/scripts/install.sh#L153

**Comment by SimFG** [comment]
If I want to use the latest, discard the changes in the lazy-lock.json file while fetching the main branch?

**Reply by ayamir** [reply]
`git restore lazy-lock.json` then `git pull`.

**Reply by SimFG** [reply]
thank you so much, i will try it
